### PR TITLE
secure-add-selinux

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [x] 21.04.x (master)

### CODE REVIEW

File: en/administration/secure-platform.md [hunk 1]
```diff
@@ -37,6 +37,134 @@ apache:x:48:48:Apache:/usr/share/httpd:/sbin/nologin
...
+directories?
+
+The standard access policy based on the user, group, and other permissions, known as Discretionary Access Control
+(DAC), does not enable system administrators to create comprehensive and fine-grained security policies, such as
+restricting specific applications to only viewing log files, while allowing other applications to append new data to
+the log files.
+
+SELinux implements Mandatory Access Control (MAC). Every process and system resource has a special security label
+called an SELinux context. A SELinux context, sometimes referred to as an SELinux label, is an identifier which
+abstractsaway the system-level details and focuses on the security properties of the entity. Not only does this provide
```
Comment (Nohzoh) [inline en/administration/secure-platform.md:63]:
```suggestion
abstracts away the system-level details and focuses on the security properties of the entity. Not only does this provides
```

File: en/administration/secure-platform.md [hunk 2]
```diff
@@ -37,6 +37,134 @@ apache:x:48:48:Apache:/usr/share/httpd:/sbin/nologin
...
+a consistent way of referencing objects in the SELinux policy, but it also removes any ambiguity that can be found in
+other identification methods. For example, a file can have multiple valid path names on a system that makes use of bind
+mounts.
+
+La politique SELinux utilise ces contextes dans une série de règles qui définissent comment les processus peuvent
+interagir entre eux et avec les différentes ressources système. Par défaut, la stratégie n'autorise aucune interaction
+à moins qu'une règle n'accorde explicitement l'accès.
```
Comment (Nohzoh) [inline en/administration/secure-platform.md:70]: French part in english documentation

File: fr/administration/secure-platform.md [hunk 3]
```diff
@@ -37,6 +37,132 @@ apache:x:48:48:Apache:/usr/share/httpd:/sbin/nologin
...
+> Ces règles sont actuellement en **mode bêta** et peuvent être activées. Vous
+> pouvez les activer en suivant cette procédure. Lors de la détection d'un
+> problème, il est possible de désactiver SELinux globalement et de nous envoyer
+> vos commentaires afin d'améliorer nos règles sur
+> [Github](https://github.com/centreon/centreon).
+
+### Présentation de SELinux
+
+Security Enhanced Linux (SELinux) fournit une couche supplémentaire de sécurité du système. SELinux répond
+fondamentalement à la question: `Le <subject> peut-il faire cette <action> sur <object> ?`, Par exemple: un serveur Web
```
Comment (Nohzoh) [inline fr/administration/secure-platform.md:54]:
```suggestion
fondamentalement à la question: `Le <suject> peut-il faire cette <action> sur <object> ?`, Par exemple: un serveur Web
```

File: fr/administration/secure-platform.md [hunk 4]
```diff
@@ -37,6 +37,132 @@ apache:x:48:48:Apache:/usr/share/httpd:/sbin/nologin
...
+SELinux implémente le contrôle d'accès obligatoire (MAC). Chaque processus et ressource système possède une étiquette
+de sécurité spéciale appelée contexte SELinux. Un contexte SELinux, parfois appelé étiquette SELinux, est un identifiant
+qui fait abstraction des détails au niveau du système et se concentre sur les propriétés de sécurité de l'entité. Non
+seulement cela fournit un moyen cohérent de référencer des objets dans la stratégie SELinux, mais cela supprime également
+toute ambiguïté qui peut être trouvée dans d'autres méthodes d'identification. Par exemple, un fichier peut avoir plusieurs
+noms de chemin valides sur un système qui utilise des montages de liaison.
+
+Pour plus d'informations à propos de SELinux, visitez la [documentation Red Hat](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/using_selinux/getting-started-with-selinux_using-selinux)
+
+### Enable SELinux in permissive mode
```
Comment (Nohzoh) [inline fr/administration/secure-platform.md:71]:
```suggestion
### Activer SELinux en mode permissif
```

File: fr/administration/secure-platform.md [hunk 5]
```diff
@@ -37,6 +37,132 @@ apache:x:48:48:Apache:/usr/share/httpd:/sbin/nologin
...
+
+Avant d'activer SELinux en **mode renforcé**, vous devez vous assurer qu'aucune erreur n'apparaît à l'aide de la
+commande suivante :
+```shell
+cat /var/log/audit/audit.log | grep -i denied
+```
+
+Si des erreurs apparaissent, vous devez les analyser et décider si ces erreurs sont régulières et doivent être ajoutées
+en plus des règles SELinux par défaut de Centreon. Pour ce faire, utilisez la commande suivante pour transformer
+l'erreur en règles SELinux:
```
Comment (Nohzoh) [inline fr/administration/secure-platform.md:153]:
```suggestion
l'erreur en règles SELinux :
```

**Review by Nohzoh — CHANGES_REQUESTED** [review]

File: en/administration/secure-platform.md [hunk 6]
```diff
@@ -37,6 +37,133 @@ apache:x:48:48:Apache:/usr/share/httpd:/sbin/nologin
...
+a consistent way of referencing objects in the SELinux policy, but it also removes any ambiguity that can be found in
+other identification methods. For example, a file can have multiple valid path names on a system that makes use of bind
+mounts.
+
+The SELinux policy uses these contexts in a series of rules which define how processes can interact with each other and
+the various system resources. By default, the policy does not allow any interaction unless a rule explicitly grants access.
+
+For more information about SELinux please see [Red Hat documentation](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/using_selinux/getting-started-with-selinux_using-selinux)
+
+### Activer SELinux en mode permissif
```
Comment (Nohzoh) [inline en/administration/secure-platform.md:73]:
```suggestion
### Activate SELinux in permissive mode
```

File: en/administration/secure-platform.md [hunk 7]
```diff
@@ -37,6 +37,133 @@ apache:x:48:48:Apache:/usr/share/httpd:/sbin/nologin
...
+   ```shell
+   yum install centreon-common-selinux \
+   centreon-broker-selinux \
+   centreon-engine-selinux \
+   centreon-gorgoned-selinux \
+   centreon-plugins-selinux
+   ```
+<!--Map server-->
+   ```shell
+   yum install centreon-map-selinux
```
Comment (Nohzoh) [inline en/administration/secure-platform.md:121]: Isn't there a dependency between map and centreon-common?

**Review by Nohzoh — CHANGES_REQUESTED** [review]

File: en/administration/secure-platform.md [hunk 8]
```diff
@@ -37,6 +37,133 @@ apache:x:48:48:Apache:/usr/share/httpd:/sbin/nologin
...
+centreon-common	0.0.10
+centreon-engine	0.0.8
+centreon-gorgoned	0.0.3
+centreon-plugins	0.0.2
+centreon-web	0.0.8
+```
+
+### Audit logs and enable SELinux
+
+Before to enable SELinux in **enforcing** mode, you need to be sure that no errors appear using the following command:
```
Comment (Nohzoh) [inline en/administration/secure-platform.md:147]:
```suggestion
Before enabling SELinux in **enforcing** mode, you need to be sure that no errors appear using the following command:
```

**Review by Nohzoh — CHANGES_REQUESTED** [review]

File: en/administration/secure-platform.md (hunk 7 above)
Comment (lpinsivy) [reply en/administration/secure-platform.md:121]: I didn't think so

**Review by Nohzoh — APPROVED** [review]

**Pull request merged**